### PR TITLE
MM-41159 - Remove free tier cloud final date to subscribe

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary

Matttermost Cloud workspaces on the legacy free tier plan will be migrated to the new Cloud Freemium coming later this year. This PR removes the message with the date they would be retired which is no longer relevant. 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-41159

#### Screenshots

**Before**

Screen Shot 2022-01-19 at 4 51 20 PM

**After**

Screen Shot 2022-01-23 at 5 55 24 PM 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -327,17 +327,15 @@
...
   "admin.billing.subscription.planDetails.productName.cloudProfessional": "Cloud Professional",
   "admin.billing.subscription.planDetails.productName.cloudStarter": "Cloud Starter",
   "admin.billing.subscription.planDetails.productName.mmCloud": "Mattermost Cloud",
   "admin.billing.subscription.planDetails.prolongedOverages": "Prolonged overages may result in additional charges.",
   "admin.billing.subscription.planDetails.seatCountOverages": "Seat count overages",
-  "admin.billing.subscription.planDetails.seeOurPlans": "See our plans",
   "admin.billing.subscription.planDetails.startDate": "Start Date: ",
   "admin.billing.subscription.planDetails.tiers.free": "Free",
-  "admin.billing.subscription.planDetails.upToXUsers": "up to {userLimit} users, until January 31st, 2022.",
+  "admin.billing.subscription.planDetails.upToXUsers": "up to {userLimit} users",
```
Comment (pablovelezvidal) [inline i18n/en.json:338]: missing end dot.

File: i18n/pl.json [hunk 2]
```diff
@@ -327,17 +327,15 @@
...
   "admin.billing.subscription.planDetails.productName.cloudProfessional": "Cloud Professional",
   "admin.billing.subscription.planDetails.productName.cloudStarter": "Cloud Starter",
   "admin.billing.subscription.planDetails.productName.mmCloud": "Chmura Mattermost",
   "admin.billing.subscription.planDetails.prolongedOverages": "Przedłużające się przekroczenia mogą spowodować naliczenie dodatkowych opłat.",
   "admin.billing.subscription.planDetails.seatCountOverages": "Przekroczenia liczby miejsc",
-  "admin.billing.subscription.planDetails.seeOurPlans": "Zobacz nasze plany",
   "admin.billing.subscription.planDetails.startDate": "Data rozpoczęcia: ",
   "admin.billing.subscription.planDetails.tiers.free": "Darmowy",
-  "admin.billing.subscription.planDetails.upToXUsers": "do {userLimit} użytkowników, do 31 stycznia 2022 roku.",
+  "admin.billing.subscription.planDetails.upToXUsers": "do {userLimit} użytkowników",
```
Comment (pablovelezvidal) [inline i18n/pl.json:338]: missing end dot.

**Review by pablovelezvidal — APPROVED** [review]
LGTM! thanks @marianunez :shipit:

File: components/admin_console/billing/billing_summary/billing_summary.tsx [hunk 3]
```diff
@@ -65,9 +65,9 @@ export const upgradeFreeTierMattermostCloud = (onUpgradeMattermostCloud: () => v
             />
         </div>
         <div className='UpgradeMattermostCloud__description'>
-            <FormattedMarkdownMessage
+            <FormattedMessage
                 id='admin.billing.subscription.upgradeMattermostCloud.description'
-                defaultMessage='The free tier is available until **January 31, 2022.** Add your payment information to continue after January 31st.'
+                defaultMessage='The free tier is limited to 10 users. Get access to more users, teams and other great features'
```
Comment (jasonblais) [inline components/admin_console/billing/billing_summary/billing_summary.tsx:70]:
```suggestion
                defaultMessage='The free tier is limited to 10 users. Get access to more users, teams and other great features.'
```

**Review by stevemudie — COMMENTED** [review]
Looks good here. I'll wait to see what comes of Nathaniel's concern before I approve it from QA.

**Comment by marianunez** [comment]
> How were the language changes generated? I see inconsistent treatment for the `upToXUsers` messages. Some of the languages have a change to the text, while others remove the string completely.

Given that the text change was to removing just the final part of the sentence I manually change the language texts and dropped the rest of the sentence but there were some languages I couldn't say for sure where the changes should happen. Officially on any text changes, we should remove the entry in the other languages so they can be re-translated. I'll do that to be consistent on all. Thanks for the call out on that 👍

**Comment by neallred** [comment]
> Officially on any text changes, we should remove the entry in the other languages so they can be re-translated

Ahh, good to know, I didn't realize that!

**Review by jasonblais — APPROVED** [review]

**Review by stevemudie — APPROVED** [review]

**Pull request merged**